src/main/java/org/folio/innreach/domain/service/impl/InnReachTransactionActionServiceImpl.java

```diff
@@ -283,12 +283,27 @@ private void updateTransactionOnLoanClaimedReturned(StorageLoanDTO loan, InnReac
     log.info("Updating patron transaction {} on the claimed returned loan {}", transaction.getId(), loan.getId());
 
     transaction.setState(CLAIMS_RETURNED);
+    removeItemHoldFieldsFromTransaction(transaction);
 
     var claimedReturnedDateSec = ofNullable(loan.getClaimedReturnedDate()).map(DateHelper::toEpochSec).orElse(-1);
 
     notifier.reportClaimsReturned(transaction, claimedReturnedDateSec);
   }
 
+  private void removeItemHoldFieldsFromTransaction(InnReachTransaction transaction) {
```

Comment:
lets move this method to `InnReachTransactionUtils` and rename it to something more generic like `clearPatronAndItemInfo`

Reply:
done